tests/data/loop/cube.csv

```diff
@@ -0,0 +1,37 @@
+#mammos csv v2
+#----------------------------------------
+# 
+# Hysteresis loop.
+# Generated from mammos_mumag 0.7.1.
+# `configuration_type` is thelist of indices referring to saved magnetized field configurations.
+# `B_ext` is the applied magnetic flux in Tesla.
+# `J` is the polarisation (in Tesla) in the direction of the applied magnetic flux.
+# `Jx`, `Jy`, `Jz` are the components of the spontaneous polarisation (in Tesla).
+# `energy_density` is the energy density (in J/m^3).
+# 
```

Comment:
Could you please re-generate this. I would assume that the two blank lines should be gone.